File: src/main/java/it/pagopa/ecommerce/dto/PaymentMethod.java
package it.pagopa.ecommerce.dto;

import lombok.Data;

import java.util.List;
import java.util.Map;

@Data
public class PaymentMethod {

    private String paymentMethodID;
    private String paymentMethodName;
    private String paymentMethodDescription;
    private String paymentMethodStatus;
    private String paymentMethodAsset;
    private List<Range> paymentMethodRanges;
    private String paymentMethodTypeCode;
    private String clientId;
    private String methodManagement;
    private Map<String, String> paymentMethodsBrandAssets;
}

File: src/main/kotlin/ecommerce/dto/PaymentMethod.kt
package ecommerce.dto

class PaymentMethod {
    var paymentMethodID: String? = null
    var paymentMethodName: String? = null
    var paymentMethodDescription: String? = null
    var paymentMethodStatus: String? = null
    var paymentMethodAsset: String? = null
    var paymentMethodRanges: List<Range?>? = null
    var paymentMethodTypeCode: String? = null
    var clientId: String? = null
    var methodManagement: String? = null
    var paymentMethodsBrandAssets: Map<String?, String?>? = null
}
